server/src/main/java/com/objectcomputing/checkins/services/guild/Guild.java

```diff
@@ -11,6 +11,9 @@
 import javax.persistence.Id;
 import javax.persistence.Table;
 import javax.validation.constraints.NotBlank;
+
+import edu.umd.cs.findbugs.annotations.Nullable;
+
```

Comment:
I think you wanted this annotation, didn't you? `io.micronaut.core.annotation.Nullable`

Reply:
Fixed import per your suggestion.